### PR TITLE
add modmail posting

### PR DESCRIPTION
**Pull request by Bobby-McBobface**

### CODE REVIEW

**Comment by Bobby-McBobface** [comment]
not tested as credentials are seemingly incorrect, pls test

File: cogs/reddit.py [hunk 1]
```diff
@@ -192,6 +195,37 @@ async def remove_entry(
...
+
+    @tasks.loop(minutes=1)
+    async def get_modmail(self):
+        new_conversations: typing.AsyncIterator[
+            asyncpraw.models.ModmailConversation
+        ] = self.subreddit.modmail.conversations(sort="unread", state="all")
+
+        async for conv in new_conversations:
+            embed = self.create_modmail_embed(conv)
+            await self.bot.modqueue_channel.send(embed=embed)
```
Comment (sebkuip) [inline cogs/reddit.py]: Please don't use the modqueue channel for modmail. Make a separate env var for the modmail channel

File: cogs/reddit.py [hunk 2]
```diff
@@ -192,6 +195,37 @@ async def remove_entry(
...
+            await conv.read()
+
+        appeals_conversations: typing.AsyncIterator[
+            asyncpraw.models.ModmailConversation
+        ] = self.subreddit.modmail.conversations(sort="unread", state="appeals")
+        async for conv in appeals_conversations:
+            embed = self.create_modmail_embed(conv)
+            await self.bot.modqueue_channel.send(embed=embed)
```
Comment (sebkuip) [inline cogs/reddit.py]: Same to other comment

**Review by sebkuip — CHANGES_REQUESTED** [review]

**Review by sebkuip — APPROVED** [review]
LGTM

**Pull request merged**